.travis.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         - mysql -e "CREATE DATABASE $TRAVIS_DB" -uroot
         - cp phinx.yml.dist phinx.yml
```

Comment:
Since this file has been removed, can we remove this line and all other occurrences?

Running

```
$ git grep phinx.yml
```

yields

```
.gitignore:25:phinx.yml
.travis.yml:68:        - cp phinx.yml.dist phinx.yml
README.md:374:2. Copy the default `phinx.yml.dist` to `phinx.yml`
script/bootstrap:23:if [ ! -f "phinx.yml" ]; then
script/bootstrap:24:    echo "Missing phinx.yml, be sure to copy from phinx.yml.dist and update config"
script/test:14:vendor/bin/phinx --configuration=phinx.yml migrate -e testing
```